airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/destination_async/StreamPriorityTest.java

```diff
@@ -35,14 +35,16 @@ void testOrderByPrioritySize() {
     when(bufferDequeue.getQueueSizeBytes(DESC2)).thenReturn(Optional.of(0L)).thenReturn(Optional.of(1L));
     final DetectStreamToFlush detect = new DetectStreamToFlush(bufferDequeue, null, new AtomicBoolean(false), null);
 
-    assertEquals(List.of(DESC1, DESC2), detect.orderStreamsByPriority(DESCS));
-    assertEquals(List.of(DESC2, DESC1), detect.orderStreamsByPriority(DESCS));
+    // DESC1 has 1 byte, DESC2 has 0 bytes
```

Comment:
I modified the existing tests to catch this. I don't think more tests are needed, however happy to add if needed.